### ISSUE TITLE
AcceptanceTester class is not getting build by build command.

### ISSUE THREAD

**Issue by hubbyist** [Codeception/Codeception#2618]
I need this functionality https://github.com/Codeception/Codeception/pull/2598 so I wanted to add them to \Acceptance\Helper but it seem as I am missing something. 

Codeception version 2.1.4

When I define these functions in \Acceptance\Helper test complains about functions are missing : Call to undefined method AcceptanceTester::stopFollowingRedirects.

file : /tests/_support/Helper/Acceptance.php
```php
namespace Helper;

// here you can define custom actions
// all public methods declared in helper class will be available in $I

class Acceptance extends \Codeception\Module {

	/**
	 * Prevents automatic redirects to be followed by the client
	 */
	public function stopFollowingRedirects(){
		$this->client->followRedirects(false);
	}

	/**
	 * Enables automatic redirects to be followed by the client
	 */
	public function startFollowingRedirects(){
		$this->client->followRedirects(true);
	}
}
```
Also it seems as if AcceptanceTester is not being build by 'build' command. Instead codecept.phar seems as if it is building NoGuy class.

```console
user@host $ php codecept.phar build
Building Actor classes for suites: functional, acceptance, unit
 -> FunctionalTesterActions.php generated successfully. 0 methods added
\FunctionalTester includes modules: \Helper\Functional
 -> NoGuyActions.php generated successfully. 0 methods added
\NoGuy includes modules: PhpBrowser, \Helper\Acceptance, REST
 -> UnitTesterActions.php generated successfully. 0 methods added
\UnitTester includes modules: Asserts, \Helper\Unit
```

When I add them to \Unit\Helper (file : /tests/_support/Helper/Unit.php) test is using functions then complains about client property is missing : Undefined property: Helper\Unit::$client.

**Comment by hubbyist** [comment]
Thanks to @janhenkgerritsen's reply at https://github.com/Codeception/Codeception/pull/2598 after changing the codes I have managed to get the desired results through UnitTester class. I added every thing in the __acceptance.suite.yml__ to __unit.suite.yml__ file and it worked as it must be in the AcceptanceTester. But codeception is still not building AcceptanceTester. Am I missing something?

**Comment by hubbyist** [comment]
When I delete NoGuy class related files : 
  /tests/_support/Helper/NoGuy.php
  /tests/_support/NoGuy.php
  /tests/_support/_generated/NoGuyActions.php
codeception is rebuilding them and not building AcceptanceTester. Any Ideas?

**Comment by hubbyist** [comment]
I found the root cause. I made a __'licp mistake'__ in __acceptance.suite.yml__. Some how I managed to delete the line :
```yaml
class_name: AcceptanceTester
```
So system start building the classes as NoGuy. So when missing line added every thing returned to normal.

__Observation:__ I tried to add another class_name parameter to see if there is a possibility of multiple classes getting build. System assumes the first class_name entry as the class name and yml file name as the folder name for the tests to be run by this class. When I copy yaml file and change the class name, two separate classes are getting build. But yaml file name is used for the second classes tests folder when run command is issued.
__Suggestion:__ Wouldn't it be simpler to use suite name (yml file name) as the class_name parameter and remove the parameter from yml files. Afterall each yaml file is producing only one class to run the test in a single directory which is the same as its file name.
